ngraph/core/src/op/gather.cpp

```diff
@@ -128,6 +128,172 @@ shared_ptr<Node> op::v1::Gather::clone_with_new_inputs(const OutputVector& new_a
     return make_shared<v1::Gather>(new_args.at(PARAMS), new_args.at(INDICES), new_args.at(AXIS));
 }
 
+NGRAPH_RTTI_DEFINITION(op::v7::Gather, "Gather", 7);
+constexpr int64_t AXIS_NOT_SET_VALUE = std::numeric_limits<int64_t>::max();
```

Comment:
Maybe `static constexpr`, not `constexpr`?

Reply:
Agree. Firstly, I didn't want to add `static` I thought it would be redundant but after your comment I did some exploration and now I got it.
![image](https://user-images.githubusercontent.com/5703039/112619499-778d6500-8e38-11eb-817e-e7ff566993e3.png)
done, please review